RevokeCgn/function.json

```diff
@@ -5,9 +5,9 @@
       "type": "httpTrigger",
       "direction": "in",
       "name": "req",
-      "route": "api/v1/cgn/{fiscalcode}",
+      "route": "api/v1/cgn/{fiscalcode}/revocation",
```

Comment:
so we'll get a neww endpoint to get revocations?
ie. GET api/v1/cgn/{fiscalcode}/revocation/{id} ?

Reply:
I got your expectation but I'd rather avoid to write multiple endpoint to reach the same resource, namely `userCgn` that is served by `/cgn/status/{fiscalcode}`. There's another 2 options:
- Rename the revocation endpoint to `api/v1/cgn/{fiscalcode}` with PUT method with no impacts on the handler's code
- Rewrite the `RevokeCgn` as `UpsertCgn` that allows to upsert `userCgn` with all the possible statuses.

IMHO the first one is better so we can keep activation and revocation separated. What do you think @gunzip ?